data_steward/cdr_cleaner/cleaning_rules/populate_survey_conduct_ext.py

```diff
@@ -0,0 +1,128 @@
+"""
+Populates survey_conduct_ext table with the language information
+as provided in the questionnaire_response_additional_info table.
+
+Original issue: DC-2627
+"""
+# Python imports
+import logging
+
+# Project imports
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from cdr_cleaner.cleaning_rules.generate_ext_tables import GenerateExtTables
+from cdr_cleaner.manual_cleaning_rules.survey_version_info import COPESurveyVersionTask
+from common import EXT_SUFFIX, JINJA_ENV, SURVEY_CONDUCT
+
+LOGGER = logging.getLogger(__name__)
+
+SANDBOX_SURVEY_CONDUCT_EXT_QUERY = JINJA_ENV.from_string("""
+CREATE TABLE `{{project_id}}.{{sandbox_dataset_id}}.{{sandbox_table}}`
+AS
+    SELECT sce.* FROM `{{project_id}}.{{dataset_id}}.survey_conduct_ext` AS sce
+    JOIN `{{project_id}}.{{dataset_id}}.questionnaire_response_additional_info` AS qrai
+    ON sce.survey_conduct_id = qrai.questionnaire_response_id
+    WHERE (sce.language IS NULL OR sce.language != qrai.value)
+    AND qrai.type = 'LANGUAGE'
```

Comment:
Because these queries are doing a string comparison, I would suggest modifying lines 26 and 34 to something like:
`AND UPPER(qrai.type) = 'LANGUAGE'`

to preserve the logic if what we receive changes case or has inconsistent case.